src/AndcultureCode.CSharp.Sitefinity.Core/Attributes/SitefinityMetadataAttribute.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace AndcultureCode.CSharp.Sitefinity.Core.Attributes
+{
+    [AttributeUsage(AttributeTargets.Class)]
+    public class SitefinityMetadataAttribute : Attribute
+    {
+        public string DynamicContentType { get; }
```

Comment:
It might be helpful to add some context by adding some commentary these two properties `DynamicContentType` and `ParentDynamicContentType`